common/tor/tor_proxy_uri_helper.cc

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/common/tor/tor_proxy_uri_helper.h"
+#include "chrome/common/channel_info.h"
+#include "components/version_info/channel.h"
+
+namespace tor {
+
+std::string GetTorProxyURI() {
+  switch (chrome::GetChannel()) {
+    case version_info::Channel::STABLE:
+      return std::string("socks5://127.0.0.1:9350");
```

Comment:
why not introduce a named constant for `127.0.0.1` and maybe ports

Reply:
fixed in 58622cb